tests/e2e/cucumber/features/integrations/link.feature

```diff
@@ -75,3 +78,7 @@ Feature: link
     # Then the public should not see the following files on the files-drop page
     #  | textfile.txt |
     And "Anonymous" logs out
+    When "Alice" logs in
```

Comment:
Please do not log in alice again, just keep her logged in the previous steps, then at the end log her out again.

Logging out is at the needed to tell playwright that the page context can be closed and de-registered.
The only reason to log-out and then re-login inside a test is when the steps between those take longer then the timeout from playwright.